2025-04-14 16:33:44,871 - cephci - log:161 - DEBUG - Completed log configuration
2025-04-14 16:33:44,874 - cephci - run:722 - INFO - Running test cephfs_nfs.nfs_export_config.py
2025-04-14 16:33:44,875 - cephci - nfs_export_config:44 - INFO - Running cephfs CEPH-83574008 test case
2025-04-14 16:33:44,876 - cephci - ceph:1576 - INFO - Execute ceph version -f json on 10.0.195.251
2025-04-14 16:33:45,880 - cephci - ceph:1606 - INFO - Execution of ceph version -f json on 10.0.195.251 took 1.003318 seconds
2025-04-14 16:33:45,880 - cephci - utils:2347 - INFO - 
{"version":"ceph version 19.2.1-126.el9cp (cfd2907537ba633f7d638895efd70ef5d0f1c99b) squid (stable)"}

2025-04-14 16:33:45,880 - cephci - utils:2350 - INFO - ceph version 19.2.1-126.el9cp (cfd2907537ba633f7d638895efd70ef5d0f1c99b) squid (stable)
2025-04-14 16:33:45,882 - cephci - ceph:1576 - INFO - Execute yum install -y --nogpgcheck python3 python3-pip fio fuse ceph-fuse attr gcc python3-devel git lua on 10.0.195.251
2025-04-14 16:33:46,886 - cephci - ceph:1186 - DEBUG - Updating Subscription Management repositories.
2025-04-14 16:33:47,528 - cephci - ceph:1186 - DEBUG - 
2025-04-14 16:33:47,528 - cephci - ceph:1186 - DEBUG - This system has release set to 9.5 and it receives updates only for this release.
2025-04-14 16:33:47,528 - cephci - ceph:1186 - DEBUG - 
2025-04-14 16:33:48,388 - cephci - ceph:1186 - DEBUG - Last metadata expiration check: 2:57:54 ago on Mon Apr 14 13:35:33 2025.
2025-04-14 16:33:48,623 - cephci - ceph:1186 - DEBUG - Package python3-3.9.21-1.el9_5.x86_64 is already installed.
2025-04-14 16:33:48,624 - cephci - ceph:1186 - DEBUG - Package python3-pip-21.3.1-1.el9.noarch is already installed.
2025-04-14 16:33:48,626 - cephci - ceph:1186 - DEBUG - Package fio-3.35-1.el9.x86_64 is already installed.
2025-04-14 16:33:48,627 - cephci - ceph:1186 - DEBUG - Package fuse-2.9.9-16.el9.x86_64 is already installed.
2025-04-14 16:33:48,628 - cephci - ceph:1186 - DEBUG - Package ceph-fuse-2:19.2.1-126.el9cp.x86_64 is already installed.
2025-04-14 16:33:48,629 - cephci - ceph:1186 - DEBUG - Package attr-2.5.1-3.el9.x86_64 is already installed.
2025-04-14 16:33:48,630 - cephci - ceph:1186 - DEBUG - Package gcc-11.5.0-5.el9_5.x86_64 is already installed.
2025-04-14 16:33:48,631 - cephci - ceph:1186 - DEBUG - Package python3-devel-3.9.21-1.el9_5.x86_64 is already installed.
2025-04-14 16:33:48,633 - cephci - ceph:1186 - DEBUG - Package git-2.43.5-2.el9_5.x86_64 is already installed.
2025-04-14 16:33:48,634 - cephci - ceph:1186 - DEBUG - Package lua-5.4.4-4.el9.x86_64 is already installed.
2025-04-14 16:33:48,725 - cephci - ceph:1186 - DEBUG - Dependencies resolved.
2025-04-14 16:33:48,726 - cephci - ceph:1186 - DEBUG - Nothing to do.
2025-04-14 16:33:48,726 - cephci - ceph:1186 - DEBUG - Complete!
2025-04-14 16:33:48,838 - cephci - ceph:1606 - INFO - Execution of yum install -y --nogpgcheck python3 python3-pip fio fuse ceph-fuse attr gcc python3-devel git lua on 10.0.195.251 took 2.955991 seconds
2025-04-14 16:33:48,881 - cephci - ceph:1576 - INFO - Execute pip3 install xattr numpy scipy on 10.0.195.251
2025-04-14 16:33:49,885 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: xattr in /usr/local/lib64/python3.9/site-packages (1.1.4)
2025-04-14 16:33:49,885 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: numpy in /usr/local/lib64/python3.9/site-packages (2.0.2)
2025-04-14 16:33:49,885 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: scipy in /usr/local/lib64/python3.9/site-packages (1.13.1)
2025-04-14 16:33:49,885 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: cffi>=1.16.0 in /usr/local/lib64/python3.9/site-packages (from xattr) (1.17.1)
2025-04-14 16:33:49,885 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: pycparser in /usr/local/lib/python3.9/site-packages (from cffi>=1.16.0->xattr) (2.22)
2025-04-14 16:33:49,885 - cephci - ceph:1186 - ERROR - WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
2025-04-14 16:33:49,886 - cephci - ceph:1606 - INFO - Execution of pip3 install xattr numpy scipy on 10.0.195.251 took 1.004635 seconds
2025-04-14 16:33:49,887 - cephci - ceph:1576 - INFO - Execute ls /home/cephuser on 10.0.195.251
2025-04-14 16:33:50,891 - cephci - ceph:1606 - INFO - Execution of ls /home/cephuser on 10.0.195.251 took 1.003583 seconds
2025-04-14 16:33:50,893 - cephci - ceph:1576 - INFO - Execute rpm -qa | grep -w 'dbench' on 10.0.195.251
2025-04-14 16:33:51,897 - cephci - ceph:1606 - INFO - Execution of rpm -qa | grep -w 'dbench' on 10.0.195.251 took 1.003924 seconds
2025-04-14 16:33:51,899 - cephci - ceph:1576 - INFO - Execute yum install -y --nogpgcheck python3 python3-pip fio fuse ceph-fuse attr gcc python3-devel git lua on 10.0.195.3
2025-04-14 16:33:52,903 - cephci - ceph:1186 - DEBUG - Updating Subscription Management repositories.
2025-04-14 16:33:53,383 - cephci - ceph:1186 - DEBUG - 
2025-04-14 16:33:53,383 - cephci - ceph:1186 - DEBUG - This system has release set to 9.5 and it receives updates only for this release.
2025-04-14 16:33:53,383 - cephci - ceph:1186 - DEBUG - 
2025-04-14 16:33:54,254 - cephci - ceph:1186 - DEBUG - Last metadata expiration check: 2:37:27 ago on Mon Apr 14 13:56:06 2025.
2025-04-14 16:33:54,493 - cephci - ceph:1186 - DEBUG - Package python3-3.9.21-1.el9_5.x86_64 is already installed.
2025-04-14 16:33:54,494 - cephci - ceph:1186 - DEBUG - Package python3-pip-21.3.1-1.el9.noarch is already installed.
2025-04-14 16:33:54,495 - cephci - ceph:1186 - DEBUG - Package fio-3.35-1.el9.x86_64 is already installed.
2025-04-14 16:33:54,497 - cephci - ceph:1186 - DEBUG - Package fuse-2.9.9-16.el9.x86_64 is already installed.
2025-04-14 16:33:54,498 - cephci - ceph:1186 - DEBUG - Package ceph-fuse-2:19.2.1-126.el9cp.x86_64 is already installed.
2025-04-14 16:33:54,499 - cephci - ceph:1186 - DEBUG - Package attr-2.5.1-3.el9.x86_64 is already installed.
2025-04-14 16:33:54,500 - cephci - ceph:1186 - DEBUG - Package gcc-11.5.0-5.el9_5.x86_64 is already installed.
2025-04-14 16:33:54,502 - cephci - ceph:1186 - DEBUG - Package python3-devel-3.9.21-1.el9_5.x86_64 is already installed.
2025-04-14 16:33:54,503 - cephci - ceph:1186 - DEBUG - Package git-2.43.5-2.el9_5.x86_64 is already installed.
2025-04-14 16:33:54,504 - cephci - ceph:1186 - DEBUG - Package lua-5.4.4-4.el9.x86_64 is already installed.
2025-04-14 16:33:54,598 - cephci - ceph:1186 - DEBUG - Dependencies resolved.
2025-04-14 16:33:54,599 - cephci - ceph:1186 - DEBUG - Nothing to do.
2025-04-14 16:33:54,599 - cephci - ceph:1186 - DEBUG - Complete!
2025-04-14 16:33:55,714 - cephci - ceph:1606 - INFO - Execution of yum install -y --nogpgcheck python3 python3-pip fio fuse ceph-fuse attr gcc python3-devel git lua on 10.0.195.3 took 3.813658 seconds
2025-04-14 16:33:55,716 - cephci - ceph:1576 - INFO - Execute pip3 install xattr numpy scipy on 10.0.195.3
2025-04-14 16:33:56,720 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: xattr in /usr/local/lib64/python3.9/site-packages (1.1.4)
2025-04-14 16:33:56,720 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: numpy in /usr/local/lib64/python3.9/site-packages (2.0.2)
2025-04-14 16:33:56,720 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: scipy in /usr/local/lib64/python3.9/site-packages (1.13.1)
2025-04-14 16:33:56,720 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: cffi>=1.16.0 in /usr/local/lib64/python3.9/site-packages (from xattr) (1.17.1)
2025-04-14 16:33:56,721 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: pycparser in /usr/local/lib/python3.9/site-packages (from cffi>=1.16.0->xattr) (2.22)
2025-04-14 16:33:56,721 - cephci - ceph:1186 - ERROR - WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
2025-04-14 16:33:56,721 - cephci - ceph:1606 - INFO - Execution of pip3 install xattr numpy scipy on 10.0.195.3 took 1.005051 seconds
2025-04-14 16:33:56,723 - cephci - ceph:1576 - INFO - Execute ls /home/cephuser on 10.0.195.3
2025-04-14 16:33:57,726 - cephci - ceph:1606 - INFO - Execution of ls /home/cephuser on 10.0.195.3 took 1.003577 seconds
2025-04-14 16:33:57,728 - cephci - ceph:1576 - INFO - Execute rpm -qa | grep -w 'dbench' on 10.0.195.3
2025-04-14 16:33:58,732 - cephci - ceph:1606 - INFO - Execution of rpm -qa | grep -w 'dbench' on 10.0.195.3 took 1.003481 seconds
2025-04-14 16:33:58,735 - cephci - ceph:1576 - INFO - Execute yum install -y --nogpgcheck python3 python3-pip fio fuse ceph-fuse attr gcc python3-devel git lua on 10.0.195.202
2025-04-14 16:33:59,740 - cephci - ceph:1186 - DEBUG - Updating Subscription Management repositories.
2025-04-14 16:34:00,256 - cephci - ceph:1186 - DEBUG - 
2025-04-14 16:34:00,257 - cephci - ceph:1186 - DEBUG - This system has release set to 9.5 and it receives updates only for this release.
2025-04-14 16:34:00,257 - cephci - ceph:1186 - DEBUG - 
2025-04-14 16:34:00,994 - cephci - ceph:1186 - DEBUG - Last metadata expiration check: 2:54:22 ago on Mon Apr 14 13:39:18 2025.
2025-04-14 16:34:01,230 - cephci - ceph:1186 - DEBUG - Package python3-3.9.21-1.el9_5.x86_64 is already installed.
2025-04-14 16:34:01,232 - cephci - ceph:1186 - DEBUG - Package python3-pip-21.3.1-1.el9.noarch is already installed.
2025-04-14 16:34:01,233 - cephci - ceph:1186 - DEBUG - Package fio-3.35-1.el9.x86_64 is already installed.
2025-04-14 16:34:01,234 - cephci - ceph:1186 - DEBUG - Package fuse-2.9.9-16.el9.x86_64 is already installed.
2025-04-14 16:34:01,235 - cephci - ceph:1186 - DEBUG - Package ceph-fuse-2:19.2.1-126.el9cp.x86_64 is already installed.
2025-04-14 16:34:01,236 - cephci - ceph:1186 - DEBUG - Package attr-2.5.1-3.el9.x86_64 is already installed.
2025-04-14 16:34:01,237 - cephci - ceph:1186 - DEBUG - Package gcc-11.5.0-5.el9_5.x86_64 is already installed.
2025-04-14 16:34:01,238 - cephci - ceph:1186 - DEBUG - Package python3-devel-3.9.21-1.el9_5.x86_64 is already installed.
2025-04-14 16:34:01,239 - cephci - ceph:1186 - DEBUG - Package git-2.43.5-2.el9_5.x86_64 is already installed.
2025-04-14 16:34:01,241 - cephci - ceph:1186 - DEBUG - Package lua-5.4.4-4.el9.x86_64 is already installed.
2025-04-14 16:34:01,332 - cephci - ceph:1186 - DEBUG - Dependencies resolved.
2025-04-14 16:34:01,333 - cephci - ceph:1186 - DEBUG - Nothing to do.
2025-04-14 16:34:01,333 - cephci - ceph:1186 - DEBUG - Complete!
2025-04-14 16:34:02,510 - cephci - ceph:1606 - INFO - Execution of yum install -y --nogpgcheck python3 python3-pip fio fuse ceph-fuse attr gcc python3-devel git lua on 10.0.195.202 took 3.774482 seconds
2025-04-14 16:34:02,511 - cephci - ceph:1576 - INFO - Execute pip3 install xattr numpy scipy on 10.0.195.202
2025-04-14 16:34:03,515 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: xattr in /usr/local/lib64/python3.9/site-packages (1.1.4)
2025-04-14 16:34:03,516 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: numpy in /usr/local/lib64/python3.9/site-packages (2.0.2)
2025-04-14 16:34:03,516 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: scipy in /usr/local/lib64/python3.9/site-packages (1.13.1)
2025-04-14 16:34:03,516 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: cffi>=1.16.0 in /usr/local/lib64/python3.9/site-packages (from xattr) (1.17.1)
2025-04-14 16:34:03,516 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: pycparser in /usr/local/lib/python3.9/site-packages (from cffi>=1.16.0->xattr) (2.22)
2025-04-14 16:34:03,516 - cephci - ceph:1186 - ERROR - WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
2025-04-14 16:34:03,516 - cephci - ceph:1606 - INFO - Execution of pip3 install xattr numpy scipy on 10.0.195.202 took 1.004669 seconds
2025-04-14 16:34:03,517 - cephci - ceph:1576 - INFO - Execute ls /home/cephuser on 10.0.195.202
2025-04-14 16:34:04,521 - cephci - ceph:1606 - INFO - Execution of ls /home/cephuser on 10.0.195.202 took 1.003668 seconds
2025-04-14 16:34:04,523 - cephci - ceph:1576 - INFO - Execute rpm -qa | grep -w 'dbench' on 10.0.195.202
2025-04-14 16:34:05,526 - cephci - ceph:1606 - INFO - Execution of rpm -qa | grep -w 'dbench' on 10.0.195.202 took 1.003387 seconds
2025-04-14 16:34:05,528 - cephci - ceph:1576 - INFO - Execute yum install -y --nogpgcheck python3 python3-pip fio fuse ceph-fuse attr gcc python3-devel git lua on 10.0.195.142
2025-04-14 16:34:06,532 - cephci - ceph:1186 - DEBUG - Updating Subscription Management repositories.
2025-04-14 16:34:07,078 - cephci - ceph:1186 - DEBUG - 
2025-04-14 16:34:07,079 - cephci - ceph:1186 - DEBUG - This system has release set to 9.5 and it receives updates only for this release.
2025-04-14 16:34:07,079 - cephci - ceph:1186 - DEBUG - 
2025-04-14 16:34:07,841 - cephci - ceph:1186 - DEBUG - Last metadata expiration check: 2:51:42 ago on Mon Apr 14 13:42:05 2025.
2025-04-14 16:34:08,075 - cephci - ceph:1186 - DEBUG - Package python3-3.9.21-1.el9_5.x86_64 is already installed.
2025-04-14 16:34:08,077 - cephci - ceph:1186 - DEBUG - Package python3-pip-21.3.1-1.el9.noarch is already installed.
2025-04-14 16:34:08,078 - cephci - ceph:1186 - DEBUG - Package fio-3.35-1.el9.x86_64 is already installed.
2025-04-14 16:34:08,079 - cephci - ceph:1186 - DEBUG - Package fuse-2.9.9-16.el9.x86_64 is already installed.
2025-04-14 16:34:08,081 - cephci - ceph:1186 - DEBUG - Package ceph-fuse-2:19.2.1-126.el9cp.x86_64 is already installed.
2025-04-14 16:34:08,082 - cephci - ceph:1186 - DEBUG - Package attr-2.5.1-3.el9.x86_64 is already installed.
2025-04-14 16:34:08,084 - cephci - ceph:1186 - DEBUG - Package gcc-11.5.0-5.el9_5.x86_64 is already installed.
2025-04-14 16:34:08,085 - cephci - ceph:1186 - DEBUG - Package python3-devel-3.9.21-1.el9_5.x86_64 is already installed.
2025-04-14 16:34:08,086 - cephci - ceph:1186 - DEBUG - Package git-2.43.5-2.el9_5.x86_64 is already installed.
2025-04-14 16:34:08,088 - cephci - ceph:1186 - DEBUG - Package lua-5.4.4-4.el9.x86_64 is already installed.
2025-04-14 16:34:08,179 - cephci - ceph:1186 - DEBUG - Dependencies resolved.
2025-04-14 16:34:08,180 - cephci - ceph:1186 - DEBUG - Nothing to do.
2025-04-14 16:34:08,180 - cephci - ceph:1186 - DEBUG - Complete!
2025-04-14 16:34:08,332 - cephci - ceph:1606 - INFO - Execution of yum install -y --nogpgcheck python3 python3-pip fio fuse ceph-fuse attr gcc python3-devel git lua on 10.0.195.142 took 2.803231 seconds
2025-04-14 16:34:08,374 - cephci - ceph:1576 - INFO - Execute pip3 install xattr numpy scipy on 10.0.195.142
2025-04-14 16:34:09,379 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: xattr in /usr/local/lib64/python3.9/site-packages (1.1.4)
2025-04-14 16:34:09,379 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: numpy in /usr/local/lib64/python3.9/site-packages (2.0.2)
2025-04-14 16:34:09,379 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: scipy in /usr/local/lib64/python3.9/site-packages (1.13.1)
2025-04-14 16:34:09,379 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: cffi>=1.16.0 in /usr/local/lib64/python3.9/site-packages (from xattr) (1.17.1)
2025-04-14 16:34:09,379 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: pycparser in /usr/local/lib/python3.9/site-packages (from cffi>=1.16.0->xattr) (2.22)
2025-04-14 16:34:09,379 - cephci - ceph:1186 - ERROR - WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
2025-04-14 16:34:09,380 - cephci - ceph:1606 - INFO - Execution of pip3 install xattr numpy scipy on 10.0.195.142 took 1.004961 seconds
2025-04-14 16:34:09,381 - cephci - ceph:1576 - INFO - Execute ls /home/cephuser on 10.0.195.142
2025-04-14 16:34:10,385 - cephci - ceph:1606 - INFO - Execution of ls /home/cephuser on 10.0.195.142 took 1.003886 seconds
2025-04-14 16:34:10,387 - cephci - ceph:1576 - INFO - Execute rpm -qa | grep -w 'dbench' on 10.0.195.142
2025-04-14 16:34:11,392 - cephci - ceph:1606 - INFO - Execution of rpm -qa | grep -w 'dbench' on 10.0.195.142 took 1.003844 seconds
2025-04-14 16:34:11,394 - cephci - ceph:1576 - INFO - Execute ceph fs ls --format json-pretty on 10.0.195.251
2025-04-14 16:34:12,398 - cephci - ceph:1606 - INFO - Execution of ceph fs ls --format json-pretty on 10.0.195.251 took 1.003856 seconds
2025-04-14 16:34:12,398 - cephci - cephfs_utilsV1:729 - INFO - Giving required permissions for clients:
2025-04-14 16:34:12,400 - cephci - ceph:1576 - INFO - Execute ceph auth get client.ceph-regression-vhr823-futhjo-node8 on 10.0.195.251
2025-04-14 16:34:13,403 - cephci - ceph:1606 - INFO - Execution of ceph auth get client.ceph-regression-vhr823-futhjo-node8 on 10.0.195.251 took 1.00361 seconds
2025-04-14 16:34:13,405 - cephci - ceph:1576 - INFO - Execute ceph auth del client.ceph-regression-vhr823-futhjo-node8 on 10.0.195.251
2025-04-14 16:34:14,409 - cephci - ceph:1606 - INFO - Execution of ceph auth del client.ceph-regression-vhr823-futhjo-node8 on 10.0.195.251 took 1.003384 seconds
2025-04-14 16:34:14,411 - cephci - ceph:1576 - INFO - Execute ceph auth get-or-create client.ceph-regression-vhr823-futhjo-node8 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node8.keyring on 10.0.195.251
2025-04-14 16:34:15,415 - cephci - ceph:1606 - INFO - Execution of ceph auth get-or-create client.ceph-regression-vhr823-futhjo-node8 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node8.keyring on 10.0.195.251 took 1.003648 seconds
2025-04-14 16:34:15,417 - cephci - ceph:1576 - INFO - Execute chmod 644 /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node8.keyring on 10.0.195.251
2025-04-14 16:34:16,420 - cephci - ceph:1606 - INFO - Execution of chmod 644 /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node8.keyring on 10.0.195.251 took 1.003079 seconds
2025-04-14 16:34:16,421 - cephci - cephfs_utilsV1:729 - INFO - Giving required permissions for clients:
2025-04-14 16:34:16,423 - cephci - ceph:1576 - INFO - Execute ceph auth get client.ceph-regression-vhr823-futhjo-node9 on 10.0.195.3
2025-04-14 16:34:17,426 - cephci - ceph:1606 - INFO - Execution of ceph auth get client.ceph-regression-vhr823-futhjo-node9 on 10.0.195.3 took 1.003398 seconds
2025-04-14 16:34:17,428 - cephci - ceph:1576 - INFO - Execute ceph auth del client.ceph-regression-vhr823-futhjo-node9 on 10.0.195.3
2025-04-14 16:34:18,432 - cephci - ceph:1606 - INFO - Execution of ceph auth del client.ceph-regression-vhr823-futhjo-node9 on 10.0.195.3 took 1.003162 seconds
2025-04-14 16:34:18,434 - cephci - ceph:1576 - INFO - Execute ceph auth get-or-create client.ceph-regression-vhr823-futhjo-node9 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node9.keyring on 10.0.195.3
2025-04-14 16:34:19,437 - cephci - ceph:1606 - INFO - Execution of ceph auth get-or-create client.ceph-regression-vhr823-futhjo-node9 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node9.keyring on 10.0.195.3 took 1.002953 seconds
2025-04-14 16:34:19,439 - cephci - ceph:1576 - INFO - Execute chmod 644 /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node9.keyring on 10.0.195.3
2025-04-14 16:34:20,443 - cephci - ceph:1606 - INFO - Execution of chmod 644 /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node9.keyring on 10.0.195.3 took 1.003231 seconds
2025-04-14 16:34:20,443 - cephci - cephfs_utilsV1:729 - INFO - Giving required permissions for clients:
2025-04-14 16:34:20,445 - cephci - ceph:1576 - INFO - Execute ceph auth get client.ceph-regression-vhr823-futhjo-node10 on 10.0.195.202
2025-04-14 16:34:21,449 - cephci - ceph:1606 - INFO - Execution of ceph auth get client.ceph-regression-vhr823-futhjo-node10 on 10.0.195.202 took 1.004175 seconds
2025-04-14 16:34:21,451 - cephci - ceph:1576 - INFO - Execute ceph auth del client.ceph-regression-vhr823-futhjo-node10 on 10.0.195.202
2025-04-14 16:34:22,455 - cephci - ceph:1606 - INFO - Execution of ceph auth del client.ceph-regression-vhr823-futhjo-node10 on 10.0.195.202 took 1.003557 seconds
2025-04-14 16:34:22,457 - cephci - ceph:1576 - INFO - Execute ceph auth get-or-create client.ceph-regression-vhr823-futhjo-node10 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node10.keyring on 10.0.195.202
2025-04-14 16:34:23,461 - cephci - ceph:1606 - INFO - Execution of ceph auth get-or-create client.ceph-regression-vhr823-futhjo-node10 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node10.keyring on 10.0.195.202 took 1.003577 seconds
2025-04-14 16:34:23,463 - cephci - ceph:1576 - INFO - Execute chmod 644 /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node10.keyring on 10.0.195.202
2025-04-14 16:34:24,467 - cephci - ceph:1606 - INFO - Execution of chmod 644 /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node10.keyring on 10.0.195.202 took 1.003572 seconds
2025-04-14 16:34:24,468 - cephci - cephfs_utilsV1:729 - INFO - Giving required permissions for clients:
2025-04-14 16:34:24,470 - cephci - ceph:1576 - INFO - Execute ceph auth get client.ceph-regression-vhr823-futhjo-node11 on 10.0.195.142
2025-04-14 16:34:25,474 - cephci - ceph:1606 - INFO - Execution of ceph auth get client.ceph-regression-vhr823-futhjo-node11 on 10.0.195.142 took 1.003708 seconds
2025-04-14 16:34:25,476 - cephci - ceph:1576 - INFO - Execute ceph auth del client.ceph-regression-vhr823-futhjo-node11 on 10.0.195.142
2025-04-14 16:34:26,481 - cephci - ceph:1606 - INFO - Execution of ceph auth del client.ceph-regression-vhr823-futhjo-node11 on 10.0.195.142 took 1.003884 seconds
2025-04-14 16:34:26,482 - cephci - ceph:1576 - INFO - Execute ceph auth get-or-create client.ceph-regression-vhr823-futhjo-node11 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node11.keyring on 10.0.195.142
2025-04-14 16:34:27,486 - cephci - ceph:1606 - INFO - Execution of ceph auth get-or-create client.ceph-regression-vhr823-futhjo-node11 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node11.keyring on 10.0.195.142 took 1.003364 seconds
2025-04-14 16:34:27,488 - cephci - ceph:1576 - INFO - Execute chmod 644 /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node11.keyring on 10.0.195.142
2025-04-14 16:34:28,492 - cephci - ceph:1606 - INFO - Execution of chmod 644 /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node11.keyring on 10.0.195.142 took 1.003534 seconds
2025-04-14 16:34:28,499 - cephci - ceph:1576 - INFO - Execute ceph nfs cluster create cephfs-nfs ceph-regression-vhr823-futhjo-node7 on 10.0.195.251
2025-04-14 16:34:29,503 - cephci - ceph:1606 - INFO - Execution of ceph nfs cluster create cephfs-nfs ceph-regression-vhr823-futhjo-node7 on 10.0.195.251 took 1.003497 seconds
2025-04-14 16:34:29,503 - cephci - cephfs_volume_management:23 - INFO - Wait for the process to start or stop
2025-04-14 16:34:29,504 - cephci - ceph:1576 - INFO - Execute ceph orch ps | grep cephfs-nfs on 10.0.195.251
2025-04-14 16:34:30,508 - cephci - ceph:1606 - INFO - Execution of ceph orch ps | grep cephfs-nfs on 10.0.195.251 took 1.00331 seconds
2025-04-14 16:34:30,509 - cephci - ceph:1576 - INFO - Execute ceph nfs cluster ls -f json on 10.0.195.251
2025-04-14 16:34:31,513 - cephci - ceph:1606 - INFO - Execution of ceph nfs cluster ls -f json on 10.0.195.251 took 1.00323 seconds
2025-04-14 16:34:31,513 - cephci - nfs_export_config:79 - INFO - ceph nfs cluster created successfully
2025-04-14 16:34:31,515 - cephci - ceph:1576 - INFO - Execute ceph fs ls --format json-pretty on 10.0.195.251
2025-04-14 16:34:32,518 - cephci - ceph:1606 - INFO - Execution of ceph fs ls --format json-pretty on 10.0.195.251 took 1.002853 seconds
2025-04-14 16:34:32,520 - cephci - ceph:1576 - INFO - Execute ceph auth get-key client.ceph-regression-vhr823-futhjo-node8 on 10.0.195.251
2025-04-14 16:34:33,524 - cephci - ceph:1606 - INFO - Execution of ceph auth get-key client.ceph-regression-vhr823-futhjo-node8 on 10.0.195.251 took 1.003777 seconds
2025-04-14 16:34:33,527 - cephci - nfs_export_config:94 - INFO - Output : AQDyOP1nSWBIGRAAS/w5RWYrXBY+OJhXD0k6FA==
2025-04-14 16:34:33,558 - cephci - ceph:1576 - INFO - Execute ceph nfs export apply cephfs-nfs -i export_file.conf on 10.0.195.251
2025-04-14 16:34:34,561 - cephci - ceph:1606 - INFO - Execution of ceph nfs export apply cephfs-nfs -i export_file.conf on 10.0.195.251 took 1.003214 seconds
2025-04-14 16:34:34,563 - cephci - ceph:1576 - INFO - Execute ceph nfs export get cephfs-nfs /export_642 on 10.0.195.251
2025-04-14 16:34:35,566 - cephci - ceph:1606 - INFO - Execution of ceph nfs export get cephfs-nfs /export_642 on 10.0.195.251 took 1.002792 seconds
2025-04-14 16:34:35,567 - cephci - nfs_export_config:144 - INFO - {
  "access_type": "RW",
  "clients": [],
  "cluster_id": "cephfs-nfs",
  "export_id": 100,
  "fsal": {
    "cmount_path": "/",
    "fs_name": "cephfs",
    "name": "CEPH",
    "user_id": "nfs.cephfs-nfs.cephfs.5d839b2a"
  },
  "path": "/",
  "protocols": [
    4
  ],
  "pseudo": "/export_642",
  "security_label": true,
  "squash": "None",
  "transports": [
    "TCP"
  ]
}

2025-04-14 16:34:35,569 - cephci - ceph:1576 - INFO - Execute mkdir -p /mnt/nfs_WMTYQ on 10.0.195.251
2025-04-14 16:34:36,572 - cephci - ceph:1606 - INFO - Execution of mkdir -p /mnt/nfs_WMTYQ on 10.0.195.251 took 1.003442 seconds
2025-04-14 16:34:36,574 - cephci - ceph:1576 - INFO - Execute mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_642 /mnt/nfs_WMTYQ on 10.0.195.251
2025-04-14 16:36:41,713 - cephci - ceph:1606 - INFO - Execution of mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_642 /mnt/nfs_WMTYQ on 10.0.195.251 took 125.138123 seconds
2025-04-14 16:36:41,713 - cephci - retry:28 - WARNING - mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_642 /mnt/nfs_WMTYQ returned mount.nfs: Connection refused
 and code 32 on 10.0.195.251, Retrying in 60 seconds...
2025-04-14 16:37:41,754 - cephci - ceph:1576 - INFO - Execute mkdir -p /mnt/nfs_WMTYQ on 10.0.195.251
2025-04-14 16:37:42,759 - cephci - ceph:1606 - INFO - Execution of mkdir -p /mnt/nfs_WMTYQ on 10.0.195.251 took 1.004834 seconds
2025-04-14 16:37:42,761 - cephci - ceph:1576 - INFO - Execute mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_642 /mnt/nfs_WMTYQ on 10.0.195.251
2025-04-14 16:39:47,897 - cephci - ceph:1606 - INFO - Execution of mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_642 /mnt/nfs_WMTYQ on 10.0.195.251 took 125.135097 seconds
2025-04-14 16:39:47,897 - cephci - retry:28 - WARNING - mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_642 /mnt/nfs_WMTYQ returned mount.nfs: Connection refused
 and code 32 on 10.0.195.251, Retrying in 120 seconds...
2025-04-14 16:41:47,978 - cephci - ceph:1576 - INFO - Execute mkdir -p /mnt/nfs_WMTYQ on 10.0.195.251
2025-04-14 16:41:48,981 - cephci - ceph:1606 - INFO - Execution of mkdir -p /mnt/nfs_WMTYQ on 10.0.195.251 took 1.003009 seconds
2025-04-14 16:41:48,983 - cephci - ceph:1576 - INFO - Execute mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_642 /mnt/nfs_WMTYQ on 10.0.195.251
2025-04-14 16:43:54,122 - cephci - ceph:1606 - INFO - Execution of mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_642 /mnt/nfs_WMTYQ on 10.0.195.251 took 125.138091 seconds
2025-04-14 16:43:54,122 - cephci - nfs_export_config:156 - ERROR - mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_642 /mnt/nfs_WMTYQ returned mount.nfs: Connection refused
 and code 32 on 10.0.195.251
2025-04-14 16:43:54,124 - cephci - nfs_export_config:157 - ERROR - Traceback (most recent call last):
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/tests/cephfs/cephfs_nfs/nfs_export_config.py", line 145, in run
    fs_util.cephfs_nfs_mount(client1, nfs_server, nfs_export_name, nfs_mounting_dir)
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/utility/retry.py", line 32, in f_retry
    return f(*args, **kwargs)
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/tests/cephfs/cephfs_utilsV1.py", line 3277, in cephfs_nfs_mount
    client.exec_command(sudo=True, cmd=command)
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/ceph/ceph.py", line 2305, in exec_command
    return self.node.exec_command(cmd=cmd, **kw)
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/ceph/ceph.py", line 1700, in exec_command
    raise CommandFailed(
ceph.ceph.CommandFailed: mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_642 /mnt/nfs_WMTYQ returned mount.nfs: Connection refused
 and code 32 on 10.0.195.251

2025-04-14 16:43:54,124 - cephci - nfs_export_config:160 - INFO - Cleaning up the system
2025-04-14 16:43:54,126 - cephci - ceph:1576 - INFO - Execute rm -rf /mnt/nfs_WMTYQ/* on 10.0.195.251
2025-04-14 16:43:55,129 - cephci - ceph:1606 - INFO - Execution of rm -rf /mnt/nfs_WMTYQ/* on 10.0.195.251 took 1.003461 seconds
2025-04-14 16:43:55,131 - cephci - ceph:1576 - INFO - Execute umount /mnt/nfs_WMTYQ on 10.0.195.251
2025-04-14 16:43:56,135 - cephci - ceph:1606 - INFO - Execution of umount /mnt/nfs_WMTYQ on 10.0.195.251 took 1.003496 seconds
2025-04-14 16:43:56,137 - cephci - run:854 - ERROR - umount /mnt/nfs_WMTYQ returned umount: /mnt/nfs_WMTYQ: not mounted.
 and code 32 on 10.0.195.251
Traceback (most recent call last):
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/run.py", line 839, in run
    rc = test_mod.run(
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/tests/cephfs/cephfs_nfs/nfs_export_config.py", line 167, in run
    client1.exec_command(sudo=True, cmd=command)
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/ceph/ceph.py", line 2305, in exec_command
    return self.node.exec_command(cmd=cmd, **kw)
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/ceph/ceph.py", line 1700, in exec_command
    raise CommandFailed(
ceph.ceph.CommandFailed: umount /mnt/nfs_WMTYQ returned umount: /mnt/nfs_WMTYQ: not mounted.
 and code 32 on 10.0.195.251
2025-04-14 16:43:56,140 - cephci - ceph:1576 - INFO - Execute podman --version | awk {'print $3'} on 10.0.195.21
2025-04-14 16:43:57,143 - cephci - ceph:1606 - INFO - Execution of podman --version | awk {'print $3'} on 10.0.195.21 took 1.002628 seconds
2025-04-14 16:43:57,144 - cephci - run:1065 - INFO - Podman Version 5.2.2
2025-04-14 16:43:57,146 - cephci - ceph:1576 - INFO - Execute docker --version | awk {'print $3'} on 10.0.195.21
2025-04-14 16:43:58,149 - cephci - ceph:1606 - INFO - Execution of docker --version | awk {'print $3'} on 10.0.195.21 took 1.00354 seconds
2025-04-14 16:43:58,151 - cephci - ceph:1576 - INFO - Execute ceph --version | awk '{print $3}' on 10.0.195.251
2025-04-14 16:43:59,155 - cephci - ceph:1606 - INFO - Execution of ceph --version | awk '{print $3}' on 10.0.195.251 took 1.003671 seconds
2025-04-14 16:43:59,156 - cephci - run:1081 - INFO - ceph Version 19.2.1-126.el9cp
2025-04-14 16:43:59,157 - cephci - run:1040 - INFO - ceph_clusters_file rerun/regression-vHr823-FUTHJO
2025-04-14 16:43:59,157 - cephci - run:927 - INFO - Test <module 'cephfs_nfs.nfs_export_config' from '/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/tests/cephfs/cephfs_nfs/nfs_export_config.py'> failed
